src/server/req_manager.c

```diff
@@ -2589,6 +2589,8 @@ mgr_node_unset(struct batch_request *preq)
 
 						default:  req_reject(rc, 0, preq);
```

Comment:
I know this isn't your code, but could you put a newline in between the default and the reject?

Reply:
Done